SPECS/systemd/systemd.spec

```diff
@@ -24,6 +24,7 @@ BuildRequires:  libgcrypt-devel
 BuildRequires:  libselinux-devel
 BuildRequires:  libxslt
 BuildRequires:  lz4-devel
+BuildRequires:  mariner-release
```

Comment:
out of curiosity, why is `mariner-release` needed as a BR?

Reply:
It provides the /etc/os-release file, which resolves this error:
```
"Program sh found: YES (/usr/bin/sh)"
"src/boot/efi/meson.build:85:24: ERROR: Command \"/usr/bin/sh -c if [ -e /etc/os-release ]; then . /etc/os-release; else . /usr/lib/os-release; fi; echo $ID\" failed with status 1."
"A full log can be found at /usr/src/mariner/BUILD/systemd-stable-254.5/build/meson-logs/meson-log.txt"
"error: Bad exit status from /var/tmp/rpm-tmp.FVYPy1 (%build)"
```

Reply:
Error log can be downloaded from [this](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=455561&view=logs&j=75cba72d-09cb-503a-3927-4ab61ac4a179&t=1118f347-dc4f-543b-ee43-a57c318710c3) 3.0-dev build.